docs/lang/articles/reference.md

```diff
@@ -214,14 +225,70 @@ is evaluated to the value of that expression.
 
 ## Simple statements
 
+A simple statement is comprised within a single logical line. Several simple statements may occur on a single line separated by semicolons. The syntax for simple statements is:
+
+```
+simple_stmt ::=  expression_stmt
+                 | assert_stmt
+                 | assignment_stmt
+                 | augmented_assignment_stmt
+                 | annotated_assignment_stmt
+                 | pass_stmt
+                 | return_stmt
+                 | break_stmt
+                 | continue_stmt
+```
+
+The following section describes the syntax of simple statements that are supported in Taichi.
+It is modeled after [the simple statements chapter of the Python language reference](https://docs.python.org/3/reference/simple_stmts.html).
+
 ### Expression statements
 
-### Assignment statements
+```
+expression_stmt    ::=  starred_expression
+starred_expression ::=  expression | (starred_item ",")* [starred_item]
+starred_item       ::=  assignment_expression | "*" or_expr
+```
+
+An expression statement evaluates the expression list (which may be a single expression).
+
+### Assignment statements {#assignment-statements}
+
+```
+assignment_stmt ::=  (target_list "=")+ (starred_expression)
+target_list     ::=  target ("," target)* [","]
+target          ::=  identifier
+                     | "(" [target_list] ")"
+                     | "[" [target_list] "]"
+                     | attributeref
+                     | subscription
+                     | "*" target
+```
+
+An assignment statement evaluates the expression list (remember that this can be a single expression or a comma-separated list,
```

Comment:
We need to explain how a Python value becomes a Taichi value through assignment statement.